package.json

```diff
@@ -1,18 +1,19 @@
 {
         "name": "mako",
         "description": "Storage backend for Manta",
-        "version": "1.0.1",
+        "version": "1.0.2",
         "author": "Joyent (joyent.com)",
         "license": "MPL-2.0",
         "private": true,
         "dependencies": {
-                "assert-plus": "0.1.1",
-                "bunyan": "0.21.4",
+                "assert-plus": "1.0.0",
+                "bunyan": "1.8.12",
                 "carrier": "0.1.7",
-                "manta": "2.0.7",
-                "moray": "git+https://github.com/joyent/node-moray.git#fd5781bc25a9bfe2ba82167664639753fb9f0ca5",
```

Comment:
Doesn't that break the bin/mako_rebalance.js script? Granted that may never again be used. ... this is in the class of those things that motivated dropping crap from mantav2.

Other than mako's tests, I think that is the only thing here using the sdcnode that ships with the mako image. I believe minnow has its own.  Was minnow working in your test?

Reply:
Minnow is working - heartbeating to moray, reflecting actual space on /manta and dealing with 1.moray.earth.example.com being rebooted. It's using its own prebuilt-node-v0.10.48-zone-fd2cc906-8938-11e3-beab-4359c665ac99 (1.6.3 sdcnode)

You're right that this breaks the mako_rebalance script :/  Maybe dropping moray back in (at the same version as minnow is using, 3.0.7) will fix it, but I'll spend time making sure I can get that rebalancing working in the first place so I know whether I've made it better or worse.